src/common/java/net/minecraftforge/gradle/common/util/RunConfig.java

```diff
@@ -196,12 +211,12 @@ private String replace(Map<String, String> vars, String value) {
         return resolved == null ? value : resolved;
     }
 
-    public static class Container {
+    public static class Container extends GroovyObjectSupport {
         private Map<String, RunConfig> runs = new HashMap<>();
```

Comment:
I guess this is the random magic that makes it actually call methodMissing? You should change the comment below to reflect that.